api/queries_pr.go

```diff
@@ -1149,6 +1152,20 @@ func BranchDeleteRemote(client *Client, repo ghrepo.Interface, branch string) er
 	return client.REST(repo.RepoHost(), "DELETE", path, nil, nil)
 }
 
+// sortPRsByPrecedence in the following order: OPEN, MERGED, CLOSED
+func sortPRsByPrecedence(prs []PullRequest) []PullRequest {
+	sort.Slice(prs, func(a, b int) bool {
+		// we only return false here if the second element has precedence than the first
+		if (prs[b].State == "MERGED" && prs[a].State =="CLOSED") || 
+			(prs[b].State == "OPEN" && prs[a].State =="CLOSED") || 
+			(prs[b].State == "OPEN" && prs[a].State == "MERGED") {
```

Comment:
Since you're only sorting by open-first (other states do not matter), I think you can get away with simply:
```go
return prs[a].State == "OPEN"
```

In this case, you might want to use SliceStable instead of Slice to preserve the original order of records that rank the same.

Reply:
Makes sense :) Updated this function, test & comment to reflect this.